typescript/browser/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
fyi at least for the omgidl repo I was told that it's fine to just have a top level repo license and it doesn't need to be included in all subsequent workspaces